File: Programming-languages-and-laboratory/Java/Exercises/19_compito_java_2018-01-29/Aziendale.java
import java.util.*;

public class Aziendale extends Cliente {

	private String ragioneSociale;
	private double portata_massima;

	public Aziendale(int id, String indirizzo, Boolean tipo_contratto, String ragioneSociale, double portata_massima) {
		super(id, indirizzo, tipo_contratto);
		this.portata_massima = portata_massima;
		this.ragioneSociale = ragioneSociale;
	}

	public String toString() {
		return id + "\taziendale\t-\t" + ragioneSociale + "\t" + super.toString() + portata_massima;
	}

	public String getNome() {
		return ragioneSociale;
	}
}

File: II - Linguaggi di programmazione e laboratorio/Esame/Python/19_compito_java_20180129/Aziendale.py
from Cliente import Cliente


class Aziendale(Cliente):
    def __init__(self, tipo, codice, ragione_sociale, portata_massima, tipo_contratto, indirizzo):
        super().__init__(tipo, codice, tipo_contratto, indirizzo)
        self._ragione_sociale = ragione_sociale
        self._portata_massima = portata_massima
        
    def get_nome(self):
        return self._ragione_sociale

    def __str__(self):
        return super().__str__() + "\t-\t" + self._ragione_sociale + "\t" + self._indirizzo + "\t" + self._tipo_contratto + "\t" + str(self._portata_massima)
